packages/components/src/Select/docs/customValue.tsx

```diff
@@ -0,0 +1,39 @@
+import { Select, Label, Avatar, Text, type ValueRenderProps } from "@hopper-ui/components";
+
+import { users, type User } from "./data.ts";
+
+const renderValue = ({ defaultChildren, selectedItem }: ValueRenderProps) => {
+    if (selectedItem) {
+        const { id, name, avatar } = selectedItem as User;
+
+        return (
+            <>
+                <Avatar key={`avatar_${id}`} name={name} src={avatar} />
+                <Text slot="label">{name}</Text>
+            </>
+        );
+    }
+
+    return defaultChildren;
+};
+
+export default function Example() {
+    return (
+        <Select 
+            fieldChildren={<Label>Users</Label>}
+            items={users}
+            renderValue={renderValue}
+            defaultSelectedKey={users[0].id}
```

Comment:
[Nit] I usually prefer accessing like this for magic arrays:

`const [firstUser] = users;`